src/lib/types/Cart.ts

```diff
@@ -24,6 +24,7 @@ export interface Cart extends Timestamps {
 		customPrice?: { amount: number; currency: Currency };
 		reservedUntil?: Date;
 		depositPercentage?: number;
+		note?: { note: string; internal: boolean };
```

Comment:
Let's use `internalNote` to store the note. That way later if we have a user note, it won't conflict

```suggestion
		internalNote?: { value: string; updatedAt: Date, updatedById: User["_id"] };
```